### PR TITLE
Issue #5 solved

### PR DESCRIPTION
**Pull request by chemphys**
#5 Finalized
@dgasmith , can you have a look?

Thanks!

### CODE REVIEW

File: src/bblock/system.cpp [hunk 1]
```diff
@@ -148,7 +150,7 @@ void System::AddMonomerInfo() {
 
 }
 
-void System::AddDimers() {
+void System::AddDimersAndTrimers() {
```
Comment (dgasmith) [inline src/bblock/system.cpp]: You might want to have this as a `add_clusters` where it takes a `n` argument depend on how you want to go.

File: src/bblock/system.cpp [hunk 2]
```diff
@@ -188,7 +191,28 @@ void System::AddDimers() {
...
         dimers_.push_back(i);
         dimers_.push_back(ret_matches[j].first);
-
+      
+     
+        // Add trimers
+        // Define query point, which is each of the points 'j' inside the 
+        // radius of 'i' 
+        double point2[3];
+        point[0] = ptc.pts[ret_matches[j].first].x;
```
Comment (dgasmith) [inline src/bblock/system.cpp]: `point2` is declared, but not used?

Comment (chemphys) [reply src/bblock/system.cpp]: that is a typo! Sorry!! I will fix it.

File: src/bblock/system.cpp (hunk 1 above)
Comment (chemphys) [reply src/bblock/system.cpp]: Will do. Thanks!

**Pull request merged**